test/fail_compilation/test16195.d

```diff
@@ -1,7 +1,7 @@
 /*
  * TEST_OUTPUT:
 ---
-fail_compilation/test16195.d(13): Error: delete p is not @safe but is used in @safe function test
+fail_compilation/test16195.d(13): Error: cannot use 'delete' in @nogc function 'test16195.test'
```

Comment:
Makes sense. I was under the impression that it would affect `scope` class, but it doesn't.
However, the following now compiles with the P.R. (commit 43ddb33):

```
@safe void dl (int* v) { delete v; }
```

Which is not the intent of this P.R.